### PR TITLE
Deadcode and XML Error cleanup

### PR DESCRIPTION
**Pull request by the-m4a**
No real "changes" here besides the alternate languages with the Diplomacy Screen stuff.  The rest is just minor things to either delete files or clean up issues that would cause XML language files to fail to load.

- Remove the dead code (Integrations\URS, Integrations\IDS)
- Move IDS text files to CQUI Text folder, renamed cqui_diplomacy_text_*
- Add diplomacy text for languages where it was missing (via google translate)
- Fix the issues with the XML files in the Integrations folders so the errors seen in Modding.log go away

### CODE REVIEW

**Comment by Infixo** [comment]
Why do you want to keep IDS text files when the IDS itself is removed? What’s the point?

**Comment by the-m4a** [comment]
image
4 strings - "Duplicate", "Trade with", "Suzerain of", and "Untradable" - which are all in the tooltips like seen above

File: Assets/UI/diplomacydealview_CQUI.lua [hunk 1]
```diff
@@ -277,11 +277,11 @@ function CQUI_renderCityButton(pCity : table, player : table, targetContainer :
 
     if pCity:IsOccupied() then
         if pCity:GetOwner() == otherPlayer:GetID() then -- Cede
-            button.IconText:SetText(button.IconText:GetText() .. '[COLOR_Civ6Green] - ' .. Locale.Lookup("LOC_IDS_DEAL_CEDE") .. '[ENDCOLOR]'); 
+            button.IconText:SetText('[COLOR_Civ6Green]' .. Locale.Lookup("LOC_DIPLOMACY_DEAL_CEDE_CITY", cityData.CityName) .. '[ENDCOLOR]');
```
Comment (Infixo) [inline Assets/UI/diplomacydealview_CQUI.lua:280]: Why this change?

Comment (the-m4a) [reply Assets/UI/diplomacydealview_CQUI.lua:280]: When I was looking at the text from IDS that was used in game, I noticed that like half the languages had the cede/return, half did not, and those that didn't would use English, I think. I then saw it's in the game anyways, and figured the trade off where the city name is now also colorized was okay (and I ended up preferring it anyway)

image

**Comment by the-m4a** [comment]
I'm going to break this one up into smaller pieces and re-do it.  Separate the "Fix text" from the "delete things we don't need"
